diff --git a/neo/SConstruct b/neo/SConstruct
index a539323..74bd584 100644
--- a/neo/SConstruct
+++ b/neo/SConstruct
@@ -313,7 +313,8 @@ elif ( BUILD == 'release' ):
 	# -finline-functions: implicit at -O3
 	# -fschedule-insns2: implicit at -O2
 	# no-unsafe-math-optimizations: that should be on by default really. hit some wonko bugs in physics code because of that
-	OPTCPPFLAGS = [ '-O3', '-march=pentium3', '-Winline', '-ffast-math', '-fno-unsafe-math-optimizations', '-fomit-frame-pointer' ]
+    # On gentoo and most modern 64-bit expecting distros, this is a hack.
+	OPTCPPFLAGS = [ '-O3', '-march=native', '-Winline', '-ffast-math', '-fno-unsafe-math-optimizations', '-fomit-frame-pointer' ]
 	if ( ID_MCHECK == '0' ):
 		ID_MCHECK = '2'
 else:
