diff -u1pr mame.orig/makefile mame/makefile
--- mame.orig/makefile	2012-01-06 03:01:16.283257655 -0600
+++ mame/makefile	2012-01-06 03:02:36.319923310 -0600
@@ -438,3 +438,2 @@ COBJFLAGS += -x objective-c++
 # this speeds it up a bit by piping between the preprocessor/compiler/assembler
-CCOMFLAGS += -pipe
 
@@ -456,3 +455,2 @@ endif
 # add the optimization flag
-CCOMFLAGS += -O$(OPTIMIZE)
 
@@ -460,13 +458,2 @@ CCOMFLAGS += -O$(OPTIMIZE)
 # and make all errors into warnings
-ifneq ($(OPTIMIZE),0)
-ifneq ($(TARGETOS),os2)
-ifndef NOWERROR
-CCOMFLAGS += -Werror -fno-strict-aliasing $(ARCHOPTS)
-else
-CCOMFLAGS += -fno-strict-aliasing $(ARCHOPTS)
-endif
-else
-CCOMFLAGS += -fno-strict-aliasing $(ARCHOPTS)
-endif
-endif
 
@@ -532,10 +519,2 @@ endif
 # flags only used when linking the core emulator
-LDFLAGS =
-ifneq ($(TARGETOS),macosx)
-ifneq ($(TARGETOS),os2)
-ifneq ($(TARGETOS),solaris)
-LDFLAGS = -Wl,--warn-common
-endif
-endif
-endif
 LDFLAGSEMULATOR =
@@ -548,7 +527,2 @@ endif
 # strip symbols and other metadata in non-symbols and non profiling builds
-ifndef SYMBOLS
-ifneq ($(TARGETOS),macosx)
-LDFLAGS += -s
-endif
-endif
 
