diff -u2pr gwibber-3.3.3.orig/gwibber/microblog/util/const.py gwibber-3.3.3/gwibber/microblog/util/const.py
--- gwibber-3.3.3.orig/gwibber/microblog/util/const.py	2012-01-30 09:12:29.000000000 -0600
+++ gwibber-3.3.3/gwibber/microblog/util/const.py	2012-02-05 02:44:19.318547184 -0600
@@ -13,6 +13,6 @@ else:
   FB_APP_KEY = "71b85c6d8cb5bbb9f1a3f8bbdcdd4b05"
 
-TWITTER_OAUTH_KEY = "qMBra1U4bpNYvDz947M5Q"
-TWITTER_OAUTH_SECRET = "Lzdkhg0WvGYFzD9tnsuwC0zYmpJ4z7HrZl3yOxU1g"
+TWITTER_OAUTH_KEY = "qp51C1PWucL45GxsYT76w"
+TWITTER_OAUTH_SECRET = "a5YnLmbuQnZTabsGm1Eg96qcMy96msVsgFLYZQ3VVGs"
 
 # Gwibber
