From 5c4e14f92dc5b3975082eddc9f3ebbfa4a4f742e Mon Sep 17 00:00:00 2001
From: Michael Muré <batolettre@gmail.com>
Date: Sat, 07 Aug 2010 12:51:02 +0000
Subject: buffer: expose gegl-buffer-iterator for use in gimp cage tool

---
diff --git a/gegl/Makefile.am b/gegl/Makefile.am
index 155758f..b4b2c43 100644
--- a/gegl/Makefile.am
+++ b/gegl/Makefile.am
@@ -36,6 +36,7 @@ GEGL_public_HEADERS =	\
     gegl-plugin.h			\
     gegl-version.h			\
     buffer/gegl-buffer.h		\
+    buffer/gegl-buffer-iterator.h	\
     property-types/gegl-paramspecs.h	\
     property-types/gegl-color.h		\
     property-types/gegl-path.h		\
--
cgit v0.8.3.1
